server/src/instant/reactive/query.clj

```diff
@@ -93,6 +96,15 @@
                      {:aggregate aggregate}))
       :child-nodes []}]))
 
+(defn- compute-instaql-topic [attrs instaql-query]
+  (tracer/with-span! {:name "compute-instaql-topic"}
```

Comment:
I do worry the extra compilation may be slow. 

I am open to ideas on what the best thing to do here is. If we compile with attrs and the instaql query, we'll likely need to be busting this cache a bunch.